charts/chainlink-cluster/devspace.yaml

```diff
@@ -25,47 +25,14 @@ pipelines:
       start_dev app                # 3. Start dev mode "app" (see "dev" section)
   deploy:
     run: |-
-      set -o pipefail
-      echo "Removing .devspace cache!"
```

Comment:
Need to confirm with @skudasov if this is needed or not.

Reply:
i dont use pipes anymore so we shouldnt need the pipefail
as for the cache, i hope we dont need it, otherwise it means that its broken :D

Reply:
It is not straightforward, but if we'll build a new image every time we can ignore the cache anyway